code/modules/uplink/uplink_item.dm

```diff
@@ -558,6 +559,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/mecha/combat/gygax/dark/loaded
 	cost = 80
 
+/datum/uplink_item/dangerous/guardian
+	name = "Holoparasites"
+	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an organic host as a home base and source of fuel."
+	item = /obj/item/weapon/storage/box/syndie_kit/guardian
+	cost = 20
+
```

Comment:
You can add this line `exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/gang)` after the cost to make it traitor gamemode only.